platforms/arm_atsam/bootloaders/md_boot.c

```diff
@@ -69,4 +73,11 @@ void bootloader_jump(void) {
     while (1)
         ;  // Wait on timeout
 }
+
+__attribute__((weak)) void keyboard_jump(void) {
+    WDT->CONFIG.bit.WINDOW   = 0;
+    WDT->CONFIG.bit.PER      = 0;
+    WDT->EWCTRL.bit.EWOFFSET = 0;
+    WDT->CTRLA.bit.ENABLE    = 1;
+}
```

Comment:
Looks like this implementation is incomplete — it starts the watchdog timer, but does not enter the infinite loop, so it might return and do something undesired while the watchdog timer is counting down.

Reply:
Honestly, I wasn't entirely sure about this, and I don't have hardware to test on.